staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/seat_seconds.go

```diff
@@ -41,6 +41,11 @@ func SeatsTimesDuration(seats float64, duration time.Duration) SeatSeconds {
 	return SeatSeconds(math.Round(seats * float64(duration/time.Nanosecond) / (1e9 / ssScale)))
 }
 
+// IsNegative tells whether the given value is effectively negative
+func (ss SeatSeconds) IsNegative() bool {
+	return ss > MaxSeatSeconds/2
```

Comment:
I don't think this would ever matter in practice but I think it should be >=, no?

Reply:
Replaced with a stricter test, no longer about "negative" or not.